README.md

```diff
@@ -480,6 +517,24 @@ headerParams:
       type: "string"
 ```
 
+#### `security`
+
+The `security` property allows you to specify the [Security Scheme](#securityschemes) to apply to the HTTP Request.  If you have applied an `overallSecurityRequirement` ([see Security on each operation](#security-on-each-operation)) then you can either leave this field off, or to override it with a different scheme you can write it like:
+
+```yml
+security:
+  - petstore_auth:
+    - write:pets
+    - read:pets
+```
+
+If you have specified an `overallSecurityRequirement`, but this HTTP Request should not apply any security schemes, you should set security to be an array with an empty object:
```

Comment:
Same as above

Reply:
Yeah i'll change it to `security`, but was mostly trying to differentiate so as to not cause confusion.